### PR TITLE
Tidying up inline code comments

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
I had an intention of creating documentation based on code comments and this still might be useful at some point, but in the meantime, fixing the actual text was still useful. Hopefully all is OK here :)

### CODE REVIEW

File: src/EventStore.ClientAPI/CatchUpSubscriptionSettings.cs [hunk 1]
```diff
@@ -3,37 +3,38 @@
 namespace EventStore.ClientAPI
 {
     /// <summary>
-    /// Settings for <see cref="EventStoreCatchUpSubscription"/>
+    /// Settings for <see cref="EventStoreCatchUpSubscription"/>.
     /// </summary>
     public class CatchUpSubscriptionSettings
     {
+        /// TODO: In what size?
```
Comment (hayley-jean) [inline src/EventStore.ClientAPI/CatchUpSubscriptionSettings.cs]: The unit here is the number of events in the queue, the same goes for the other TODO further on.

Comment (ChrisChinchilla) [reply src/EventStore.ClientAPI/CatchUpSubscriptionSettings.cs]: @hayley-jean Updated :)

**Comment by hayley-jean** [comment]
Thanks @ChrisChinchilla ! 👍 

There are just a couple of typos failing the build with invalid xml tags in `IEventStoreConnection.cs`
From the build logs, it appears that the culprits are :
- ConditionalAppendToStream
- ReadStreamEventsForwardAsync
- SubscribeToStreamAsync

**Comment by ChrisChinchilla** [comment]
Sorry about that @hayley-jean should be fixed now.

File: src/EventStore.ClientAPI/IEventStoreConnection.cs [hunk 2]
```diff
@@ -49,111 +50,114 @@ public interface IEventStoreConnection : IDisposable
         Task<DeleteResult> DeleteStreamAsync(string stream, long expectedVersion, UserCredentials userCredentials = null);
 
         /// <summary>
-        /// Deletes a stream from the Event Store asynchronously
+        /// Deletes a stream from Event Store asynchronously.
         /// </summary>
         /// <param name="stream">The name of the stream to delete.</param>
         /// <param name="expectedVersion">The expected version that the streams should have when being deleted. <see cref="ExpectedVersion"/></param>
         /// <param name="hardDelete">Indicator for tombstoning vs soft-deleting the stream. Tombstoned streams can never be recreated. Soft-deleted streams
+        /// TODO: Can the below be a link?
```
Comment (hayley-jean) [inline src/EventStore.ClientAPI/IEventStoreConnection.cs]: If you mean adding an external link, you can do that with an `<a>`, for example: `<a href="link">link text</a>`.
The link will show in intellisense, and docFx should be able to parse it when it's compiled.

File: src/EventStore.ClientAPI/IEventStoreConnection.cs [hunk 3]
```diff
@@ -49,111 +50,114 @@ public interface IEventStoreConnection : IDisposable
...
         /// can be written to again, but the EventNumber sequence will not start from 0.</param>
         /// <param name="userCredentials">The optional user credentials to perform operation with.</param>
         /// <returns>A <see cref="Task&lt;DeleteResult&gt;"/> containing the results of the delete stream operation.</returns>
         Task<DeleteResult> DeleteStreamAsync(string stream, long expectedVersion, bool hardDelete, UserCredentials userCredentials = null);
 
+        /// TODO: Differences?
```
Comment (hayley-jean) [inline src/EventStore.ClientAPI/IEventStoreConnection.cs]: The three `AppendToStreamAsync` methods are essentially the same, they just have different signatures.

File: src/EventStore.ClientAPI/IEventStoreConnection.cs [hunk 4]
```diff
@@ -176,43 +180,44 @@ public interface IEventStoreConnection : IDisposable
...
+        /// <param name="start">The position to start reading from.</param>
+        /// <param name="count">The count to read from the position.</param>
+        /// <param name="resolveLinkTos">Whether to resolve LinkTo events automatically.</param>
         /// <param name="userCredentials">The optional user credentials to perform operation with.</param>
         /// <returns>A <see cref="Task&lt;StreamEventsSlice&gt;"/> containing the results of the read operation.</returns>
         Task<StreamEventsSlice> ReadStreamEventsBackwardAsync(string stream, long start, int count, bool resolveLinkTos, UserCredentials userCredentials = null);
 
         /// <summary>
-        /// Reads All Events in the node forward asynchronously (e.g. beginning to end)
+        /// TODO: Have we defined node before as a typical word?
```
Comment (hayley-jean) [inline src/EventStore.ClientAPI/IEventStoreConnection.cs]: `node` should be common enough for anyone who has got this far to understand its meaning. In order to have a connection to perform this on, you would have had to have connected either to a single node or a cluster.

**Comment by hayley-jean** [comment]
Thanks @ChrisChinchilla !
I noticed that I'd missed a good few questions you asked in TODO's, I've added a review to respond to them. Let me know if anything's not clear

**Pull request merged**